src/pybind/mgr/orchestrator/module.py

```diff
@@ -310,7 +310,7 @@ def _list_devices(self, hostname=None, format='plain', refresh=False):
             out = []
 
             table = PrettyTable(
-                ['HOST', 'PATH', 'TYPE', 'SIZE', 'DEVICE', 'AVAIL',
+                ['HOST', 'PATH', 'TYPE', 'SIZE', 'DEVICE_ID', 'MODEL', 'VENDOR', 'ROTATIONAL', 'AVAIL',
```

Comment:
uah. do we need a `wide` mode? See [`-o wide`](https://kubernetes.io/docs/reference/kubectl/overview/#syntax-1).

Reply:
or we shrink the length of the `REJECTED REASON` field, which takes about 40% of the entire column.